jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildDockerMojo.java

```diff
@@ -48,7 +49,14 @@ public void execute() throws MojoExecutionException {
 
     // Parses 'from' and 'to' into image reference.
     ImageReference baseImage = parseBaseImageReference(getBaseImage());
-    ImageReference targetImage = parseTargetImageReference(getTargetImage());
+
+    // Parses and validates 'to'.
+    String targetImageString = getTargetImage();
+    if (Strings.isNullOrEmpty(targetImageString)) {
+      // Use default value "project-name:project-version"
+      targetImageString = getProject().getName() + ":" + getProject().getVersion();
```

Comment:
I think for this we could just directly build the `ImageReference` rather than passing it through the parser. However, we do need to be careful if the name or version is not in the valid format for a repository or tag. This can just be a TODO for now, but we should take care of these cases in the future.